db/migrate/20200511043818_add_more_flair_columns_to_group.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class AddMoreFlairColumnsToGroup < ActiveRecord::Migration[6.0]
+  def change
+    add_column :groups, :flair_icon, :string
+    add_reference :groups, :flair_image, foreign_key: { to_table: :uploads }
+
+    DB.exec(
+      <<~SQL
+        UPDATE groups SET flair_icon = flair_url
+        WHERE flair_url LIKE 'fa%'
+      SQL
+    )
+
+    DB.exec(
+      <<~SQL
+        UPDATE groups g1
+        SET flair_image_id = u.id
+        FROM groups g2
+        INNER JOIN uploads u ON g2.flair_url ~ CONCAT('\/', u.sha1, '[\.\w]*')
+        WHERE g1.id = g2.id
+      SQL
+    )
+
+    DB.exec(
+      <<~SQL
+        UPDATE groups SET flair_url = NULL
```

Comment:
I think we should be careful here, this erases `flair_url` columns when if you revert the migration for some reason. It might be safer to use `up` `down` methods instead.